lib/Runtime/Library/JavascriptString.cpp

```diff
@@ -231,8 +231,17 @@ namespace Js
             return nullptr;
         }
 
+        ScriptContext *scriptContext = GetScriptContext();
         Js::PropertyRecord const * propertyRecord;
-        GetScriptContext()->GetOrAddPropertyRecord(GetString(), GetLength(), &propertyRecord);
+        scriptContext->GetOrAddPropertyRecord(GetString(), GetLength(), &propertyRecord);
+
+        PropertyString * propertyString = scriptContext->TryGetPropertyString(propertyRecord->GetPropertyId());
```

Comment:
this is a very cheap call. Hoping it won't affect perf badly for micro benchmarks